dev/io.openliberty.io.smallrye.openapi.jaxrs/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: io.openliberty.io.smallrye.openapi.jaxrs; singleton:=true
+
+WS-TraceGroup: MPOPENAPI
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+src: src
+
+-dsannotations-inherit: true
+
+Import-Package: \
+  *
+
+Export-Package: \
+  io.smallrye.openapi.jaxrs;thread-context=true;version=2.0.0
```

Comment:
This was based on our Slack conversation from May 19th: https://ibm-cloud.slack.com/archives/D012T40FU9M/p1589902337004200

However, I notice that the relevant API in `AnnotationScannerFactory` has not been modified so that the ClassLoader can be passed in.  I assume that this means we no longer need to specify `thread-context=true` for this package.  I will make the change and run all of the tests to see if works.

Reply:
In `ApplicationProcessor` you're not passing in any classloader, so I think it's still going to use the TCCL, though I'm not sure what that will actually be at this point since I think this is called during application startup. https://github.com/OpenLiberty/open-liberty/pull/13589/files#diff-2faebd09c1f5caa512f7b9e2ef55525cR354

I would want to check what the JAX-RS annotation processor does and whether it's working as I can't see how the current code would be able to load it.